storage/src/qmdb/any/ordered/mod.rs

```diff
@@ -834,7 +834,8 @@ impl<
         C: Contiguous<Item = Operation<K, V>>,
         I: Index<Value = Location>,
         H: Hasher,
-    > crate::store::Store for Db<E, C, I, H, Update<K, V>>
+        S: State<DigestOf<H>>,
+    > crate::store::Store for Db<E, C, I, H, Update<K, V>, S>
```

Comment:
This seemed to be an oversight from before -- we should have always supported get() on both clean & dirty!